integration-tests/smoke/automation_test.go

```diff
@@ -1248,6 +1248,9 @@ func TestSetOffchainConfigWithMaxGasPrice(t *testing.T) {
 				require.NoError(t, err, "Error setting upkeep offchain config")
 			}
 
+			// sleep for 10s to make sure any in-flight performs are done
+			time.Sleep(10 * time.Second)
```

Comment:
is there a better way for performs to get done instead of hardocded time.sleep

Reply:
Do we have a number of performs we expect that we can wait for using `gom.Eventually` like above

Reply:
it's difficult to expect how many times it will perform so i use the n + 1 check from other tests to account for 1 possible in-flight request